membership_extension/__manifest__.py

```diff
@@ -1,14 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
 # Copyright 2017 David Vidal <david.vidal@tecnativa.com>
 # Copyright 2017 Luis M. Ontalba <luis.martinez@tecnativa.com>
 # Copyright 2017-2018 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2019 Alberto Martín Cortada <alberto.martin@guadaltech.es>
+
```

Comment:
Thanks @amcor !

Being picky, you should not put your copyright for this migration (no code or functionality was added) :slightly_smiling_face: 

Also don't add unnecessary blank lines

Reply:
No problem! in the collaborators readme it's correct ?

Reply:
Sure! :slightly_smiling_face: